### PR TITLE
You forgot to add contributors!

### PR DESCRIPTION
**Pull request by rickychien**
@EragonJ, could you take some time to review this? thanks!

### CODE REVIEW

File: src/frontend/js/modules/EasterEggs.js [hunk 1]
```diff
@@ -31,7 +31,8 @@ define(function(require) {
       '%E5%9B%A0%E7%82%BA%E5%A6%B3%E5%B0%B1%E6%98%AF%E6%88%91%E6%89%BE%E5%B0%8B%2025%20%E5%B9%B4%E7%9A%84%E9%82%A3%E9%A1%86%E9%81%BA%E8%90%BD%E4%B9%8B%E6%98%9F',
       '%E6%9C%80%E5%BE%8C%EF%BC%8C%E9%80%81%E7%B5%A6%E5%A6%B3%E9%80%99%E9%A6%96%E5%B1%AC%E6%96%BC%E6%88%91%E5%80%91%E5%85%A9%E5%80%8B%E4%BA%BA%E7%9A%84%E6%AD%8C',
       'Adam%20Levine%20-%20Lost%20Stars',
-      '%E7%94%9F%E6%97%A5%E5%BF%AB%E6%A8%82%20:)'
+      '%E7%94%9F%E6%97%A5%E5%BF%AB%E6%A8%82%20:)',
+      "%E6%9C%80%E5%BE%8C%EF%BC%8C%E4%B8%8D%E8%A6%81%E5%BF%98%E4%BA%86%E9%82%84%E6%9C%89%E8%83%8C%E5%BE%8C%E8%BE%9B%E8%8B%A6%E7%9A%84%E8%B2%A2%E7%8D%BB%E8%80%85%E5%80%91%20mimi89999,%20rickychien,%20mpizza"
```
Comment (crh0716) [inline src/frontend/js/modules/EasterEggs.js]: Suggest to add `%20crh0716`. What do you think?

Comment (cctuan) [inline src/frontend/js/modules/EasterEggs.js]: plz also add %20cctuan .

Comment (rickychien) [inline src/frontend/js/modules/EasterEggs.js]: @mpizza Hey! What do you think?

**Comment by EragonJ** [comment]
Nooooooo xD

**Pull request closed without merging**